Create the udev rules file for the ASI filter wheel
--- 70-asi-fw.rules	1970-01-01 01:00:00.000000000 +0100
+++ 70-asi-fw.rules	2018-09-25 00:18:43.247858211 +0000
@@ -0,0 +1,26 @@
+#
+# 70-asi-fw.rules -- Udev rules for ZWO ASI filter wheel
+#
+# Copyright 2018,2019 James Fidell (james@openastroproject.org)
+#
+# License:
+#
+# This file is part of the Open Astro Project.
+#
+# The Open Astro Project is free software: you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# The Open Astro Project is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with the Open Astro Project.  If not, see
+# <http://www.gnu.org/licenses/>.
+#
+
+# There is only one
+SUBSYSTEM=="usb", ATTR{idVendor}=="03c3", ATTR{idProduct}=="1f01", TAG+="uaccess"
